bin/internal/shared.sh

```diff
@@ -254,15 +254,15 @@ function shared::execute() {
   upgrade_flutter 7< "$SHARED_NAME"
 
   case "$BIN_NAME" in
-    flutter*)
+    flutter-dev*)
```

Comment:
nit, can you at least remove the star glob from this one?

Reply:
Okie dokie